doc/source/notebooks/advanced/multiclass_classification.py

```diff
@@ -18,10 +18,10 @@ def plot_posterior_predictions(m, X, Y):
     xx = np.linspace(X.min(), X.max(), 200).reshape(-1,1)
```

Comment:
Multiclass classification notebook doesn't have MCMC, right?

Reply:
not entirely sure what you mean but doesnt have mcmc, but effectively the mcmc notebook uses some plotting functionality from multiclass_classification

Reply:
got it, I thought it is a notebook